src/grt/grt-wave_options-parse-debug.adb

```diff
@@ -0,0 +1,58 @@
+--  GHDL Run Time (GRT) - mono-thread version.
+--  Copyright (C) 2005 - 2014 Tristan Gingold
```

Comment:
Do you mean that I should write "Copyright (C) 2016 Jonas Baggett" ?
What do I need to change in the first line ?